contracts/truefi2/LineOfCreditAgency.sol

```diff
@@ -727,6 +729,8 @@ contract LineOfCreditAgency is UpgradeableClaimable, ILineOfCreditAgency {
         address borrower,
         uint256 totalInterest
     ) internal {
+        require(bucketNumber > 0, "LineOfCreditAgency: Bucket of index 0 is not supported");
```

Comment:
Added this check for extra safety. It seems redundant, and I don't see a way to trigger it. Also that's why I didn't test it.